### ISSUE TITLE
Bus factor of 1

### ISSUE THREAD

**Issue by daviddrysdale** [daviddrysdale/python-phonenumbers#274]
This project has a sole maintainer (@daviddrysdale), and has had since the project inception in 2011.

This means that new releases are entirely gated on the availability of the maintainer.  More seriously, there is no transition plan if the maintainer becomes unavailable more permanently.

Need to think about how to deal, in the (hopefully unlikely) circumstance that this becomes relevant.

**Comment by jlaine** [comment]
Hi David,

I feel your pain, having the same problem for several of my projects! I think @rbarrois and myself could help out here. We use `phonenumbers` as part of our job and both have experience maintaining Python packages, e.g.:

https://github.com/FactoryBoy/factory_boy
https://github.com/rbarrois/python-semanticversion
https://github.com/aiortc/aiortc
https://github.com/aiortc/aioquic

Let us know if we can help out.

Take care,
Jeremy

**Comment by AA-Turner** [comment]
I'm also happy to help if useful as an (infrequent) past contributor -- I currently maintain Sphinx/Docutils so very happy to support ``phonenumbers``.

Thanks,
Adam

**Comment by jlaine** [comment]
@AA-Turner has a point, @rbarrois and I have never contributed to this repo so in your shoes I'd have a healthy suspicion of new contributors offering to co-maintain. The offer still stands if you want help further down the line.

**Comment by msaelices** [comment]
@daviddrysdale, we are using `phonenumbers` in my company for many use cases. Recently, Cython released the 3.0 version with a [Pure Python mode](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html), and I think it would be a game changer in this library, as most of the code is just Python logic using basic types. I would may be interested in helping migrate part of the code to Cython 3.0 and drop support for Python 2.

What do you think about it?

**Comment by daviddrysdale** [comment]
@msaelices : could you raise your suggestion as a separate issue please, so any discussion can be separated from this one &ndash; thanks.

**Comment by msaelices** [comment]
> @msaelices : could you raise your suggestion as a separate issue please, so any discussion can be separated from this one – thanks.

Sure: https://github.com/daviddrysdale/python-phonenumbers/issues/283
